src/todo_list/todo_list_db.py

```diff
@@ -0,0 +1,59 @@
+from postgres import with_postgres_connection
+from task import Task
+
+
+@with_postgres_connection
+def insert_row(cursor, task_name, operation_name="inserted", table_name="todo_list"):
+    insert_entry = 'insert into todo_list (task_name) values (%s)'
+    cursor.execute(insert_entry, (task_name,))
+    return None
+
+
+@with_postgres_connection
+def find_task_entries_for_task_name(cursor, task_name, operation_name="found (all)", table_name="todo_list"):
+    try:
+        query = 'select * from todo_list where task_name = %s'
+        cursor.execute(query, (task_name,))
+        records = cursor.fetchall()
+        if records is None:
+            raise Exception("Error: record for task {} does not exist".format(task_name))
+    except (Exception, psycopg2.Error) as error:
+        return None, error
+    return records, None
+
+
+@with_postgres_connection
+def find_task_entry_for_task_id(cursor, idx, operation_name="found", table_name="todo_list"):
+    try:
+        query = 'select * from todo_list where idx = %s'
```

Comment:
Does `%d` work? psycopq figure out the type on its own

Reply:
looking at the documentation, variables placeholder has to always be %s even if %d / %f makes mores sense